<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab Recorder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_master" rowstretch="0,1" columnstretch="10,15">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_recording">
      <property name="title">
       <string>Recording Control</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_location">
      <property name="title">
       <string>File Location</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="locationEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="locationLabel"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_streams">
      <property name="title">
       <string>Record from Streams</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_streams">
       <item>
        <widget class="QListWidget" name="streamList">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_streamsel">
         <item>
          <widget class="QPushButton" name="selectAllButton">
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectNoneButton">
           <property name="text">
            <string>Select None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Update</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+U</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_locationBuilder">
      <property name="title">
       <string>Location Builder</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_locationBuilder">
       <property name="title" stdset="0">
        <string>Location Builder</string>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_rootFolder">
         <property name="title">
          <string>Study Root</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_studyRoot">
          <item>
           <widget class="QLineEdit" name="rootEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="rootBrowseButton">
            <property name="text">
             <string>Browse...</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+B</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_blockTask">
         <item>
          <widget class="QLabel" name="label_currBlock">
           <property name="text">
            <string>Block/Task (%b):</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_storage">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_legacy">
          <attribute name="title">
           <string>Legacy Builder</string>
          </attribute>
          <widget class="QWidget" name="legacyLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>261</width>
             <height>66</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="legacy_layout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_expNum">
              <property name="text">
               <string>Exp num (%n)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="experimentNumberSpin">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_bids">
          <attribute name="title">
           <string>BIDS Builder</string>
          </attribute>
          <widget class="QWidget" name="bidsLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>261</width>
             <height>126</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="bids_layout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_participant">
              <property name="text">
               <string>Participant (%p)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_participant"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_session">
              <property name="text">
               <string>Session (%s)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_session"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_acq">
              <property name="text">
               <string>Acq. (%a)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_acq"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_run">
              <property name="text">
               <string>Run (%r)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBox_run">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadConfig"/>
    <addaction name="actionSaveConfig"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadConfig">
   <property name="text">
    <string>&amp;Load configuration...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="text">
    <string>&amp;Save configuration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
